posthog.go

```diff
@@ -172,6 +176,25 @@ func (c *client) Enqueue(msg Message) (err error) {
 	case Capture:
 		m.Type = "capture"
 		m.Timestamp = makeTimestamp(m.Timestamp, ts)
+		if m.SendFeatureFlags {
```

Comment:
Ah, I see why!

And this is here because capture doesn't have access to the client. Hmm, this library interface is annoying, with all that enqueuing happening by the client, vs handled internally.

I guess this is fine :)

I'd only ask to remove `sendFeatureFlags` as a key from the message here!

Reply:
I can't delete the field directly from the Message object (bc Go says I can't) but I can omit this in the capture API-ify message which should do the same thing!